internal/jobs/launch_instance_aws.go

```diff
@@ -60,6 +61,12 @@ func HandleLaunchInstanceAWS(ctx context.Context, job *worker.Job) {
 
 	jobErr = DoLaunchInstanceAWS(ctx, &args)
 
+	if jobErr != nil {
```

Comment:
Can we move this to the above statement so it is clear we are checking the error of that statement? It's common practice not to divide by new line for these cases.

OTH I'd add new line after this if, as the following statement is not related :)